.editorConfig

```diff
@@ -0,0 +1,30 @@
+# EditorConfig file: http://EditorConfig.org
+
+# Top-most EditorConfig file.
+root = true
+
+# Unix-style newlines with a newline ending every file.
+[*]
+end_of_line = lf
+insert_final_newline = true
+charset = utf-8
+trim_trailing_whitespace = true
+indent_style = space
+
+# 2 space indentation for configuration files.
+[*.{yml,config, src}]
+indent_size = 2
```

Comment:
Just to be certain, is it safe to say all files pretty much with use 4 spaces? I guess Sublime auto detected the proper spacing for config files differently, which I suppose messed me up.